challenge.md

```diff
@@ -4,22 +4,32 @@
 
 Write your answers inside this file, where it's indicated by the comments.
 
+
+
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
+Make sure you're in the right local folder, then use command git clone with SSH link, then make sure you're in the branch you want ot be in.
+
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
+git diff HEAD
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
 
+git log
+
 4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
 
+You would create a new branch using git checkout -b and then the name of the new branch.
 
-5. I want to add a file to the staging area. What git commit would I use?
+5. I want to add a file to the staging area. What git command would I use?
 
+git mv
 
 6. I've been working on a branch, add-directories, to implement a feature. The feature is done and you need to add your changes into the master branch. What git command should I use?
 
-
+git commit
+git push
```

Comment:
In order to get your changes on the master branch, you want to merge your add-directories branch into your master branch.

First, you change branch to master:

```bash
git checkout master
```

You then merge another branch into master:

```bash
git merge add-directories
```